### ISSUE TITLE
How to using pointnet without seg?

### ISSUE THREAD

**Issue by francoe0088** [fxia22/pointnet.pytorch#57]
Thanks for help,I want using pointnet for classification with new data, but data only have points,it didn't have points_label and seg_img, can pointnet classification  without points_label and seg_img?

**Comment by dhirajsuvarna** [comment]
Yes you can used it, check the flags present in the train_classification.py
